lib/frisby.js

```diff
@@ -725,21 +725,21 @@ Frisby.prototype.expectJSON = function(jsonTest) {
   return this;
 };
 
-
 // Helper to validate JSON response against provided JSONSchema structure or document
-Frisby.prototype.expectJSONSchema = function(path, jsonSchema) {
+Frisby.prototype.expectJSONSchema = function(path, jsonSchema, context) {
   var self       = this,
       args       = Array.prototype.slice.call(arguments),
-      path       = typeof args[0] === 'string' && args.shift(),
-      jsonSchema = (typeof args[0] === 'object' || typeof args[0] === 'string') && args.shift();
```

Comment:
By changing this, you are subtly changing the API of this method. You fixed it in the tests in this project, but what about everyone using this `expectJSONSchema` method in the wild already? Now they can't just use the schema without a path like before, and this will cause their code to break.